### PR TITLE
gpu-panda and tutorial ipynb

### PR DESCRIPTION
**Pull request by dcolinmorgan**
not sure how 4 files got in here, just wanted panda.py and tutorial, can you just delete other 2?

### CODE REVIEW

File: netZooPy/panda/panda.py [hunk 1]
```diff
@@ -3,21 +3,21 @@
 import math
 import time
 import pandas as pd
-import numpy as np
 from scipy.stats import zscore
-from .timer import Timer
+# from .timer import Timer
```
Comment (marouenbg) [inline netZooPy/panda/panda.py:7]: @dcolinmorgan why did you comment this line?

File: netZooPy/panda/panda.py [hunk 2]
```diff
@@ -79,9 +79,12 @@ def __init__(self, expression_file, motif_file, ppi_file, save_memory = False, s
         # =====================================================================
         # Running PANDA algorithm
         # =====================================================================
+        # if computing=='gpu' and self.motif_data is not None:
+        #     print('Running GPU PANDA algorithm ...')
+        #     self.panda_network = self.GPU_panda_loop(self.correlation_matrix, self.motif_matrix, self.ppi_matrix)
```
Comment (marouenbg) [inline netZooPy/panda/panda.py:84]: could you please delete the three commented lines here

File: netZooPy/panda/panda.py [hunk 3]
```diff
@@ -420,32 +423,68 @@ def update_diagonal(diagonal_matrix, num, alpha, step):
...
+        
         while hamming > 0.001:
             # Update motif_matrix
-            W = 0.5 * (t_function(ppi_matrix, motif_matrix) + t_function(motif_matrix, correlation_matrix))  # W = (R + A) / 2
-            hamming = np.abs(motif_matrix - W).mean()
-            motif_matrix *= (1 - alpha)
-            motif_matrix += (alpha * W)
-
-            if hamming > 0.001:
+            if computing=='gpu':
```
Comment (marouenbg) [inline netZooPy/panda/panda.py:452]: pretty cool

File: netZooPy/panda/panda.py [hunk 4]
```diff
@@ -420,32 +423,68 @@ def update_diagonal(diagonal_matrix, num, alpha, step):
...
+                motif = gt_function(motif_matrix.T)
                 # Update ppi_matrix
+                gupdate_diagonal(ppi, num_tfs, alpha, step)
+                ppi_matrix *= (1 - alpha)
+                ppi_matrix += (alpha * ppi)
+                # Update correlation_matrix
+                gupdate_diagonal(motif, num_genes, alpha, step)
+                correlation_matrix *= (1 - alpha)
+                correlation_matrix += (alpha * motif)
+            else:
```
Comment (marouenbg) [inline netZooPy/panda/panda.py:471]: else if computing=='cpu' 
it is better to specify the condition

File: netZooPy/panda/panda.py (hunk 1 above)
Comment (dcolinmorgan) [reply netZooPy/panda/panda.py:7]: oh damn

Comment (dcolinmorgan) [reply netZooPy/panda/panda.py:7]: ok i will delete this PR and try again :(

**Pull request closed without merging**